### PR TITLE
Standardise the UG

### PR DESCRIPTION
**Pull request by bryanhce**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,7 +186,7 @@ This command is typically used to **check a guest in** to your hotel with their
 
 **Constraints of parameters**:
 
-&nbsp;&nbsp;&nbsp;Unless stated otherwise, all fields should not be left blank.
+&nbsp;&nbsp;&nbsp;Unless stated otherwise, all fields **should not** be left blank.
 * `NAME` should only contain alphanumeric characters and spaces.
 * `PHONE` should only contain numbers, and it should be at least 3 digits and at most 15 digits long.
 * `EMAIL` should only contain alphanumeric characters and '@' character. The local segment (before "@") should be at most 64 characters long, and the domain segment (after "@") should be at most 255 characters long.
```
Comment (kangzongxian) [inline docs/UserGuide.md:192]: Should we delete the 'a hyphen' for the ROOM part below as we allow more than 1 hyphen? It has been mentioned in the next sentence.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -256,20 +270,22 @@ the rooms that have room clean statuses set to `no`.
...
 **Common mistakes**:
-* Searching for guests using incomplete keywords e.g. `170` will **not** match `170.00`.
+* Searching for guests using incomplete keywords e.g. `170` will **not match** `170.00`.
 
 ### Deleting a guest: `delete`
 
-This command deletes the specified guest _(with index in different scenario)_ from GuestBook. 
+This command deletes an existing guest in GuestBook.
+
+The guest specified at the `INDEX` is the guest you wish to delete.
```
Comment (kangzongxian) [inline docs/UserGuide.md:279]: Should we specify that it is from 'the current list'?

**Review by kangzongxian — CHANGES_REQUESTED** [review]
Looks good to me except for some minor changes!

File: docs/UserGuide.md (hunk 1 above)
Comment (bryanhce) [reply docs/UserGuide.md:192]: Yes!

File: docs/UserGuide.md (hunk 2 above)
Comment (bryanhce) [reply docs/UserGuide.md:279]: Yes

**Review by kangzongxian — APPROVED** [review]
Looks good to me, great work!

**Pull request merged**